exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,15 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
```

Comment:
Well done! 
Code works perfectly!
You could however drop the last `if `statement on line 13 and it would work as well, and be clearer to read